jetty9-base/src/main/assembly/assembly.xml

```diff
@@ -33,12 +33,5 @@
         <exclude>**/META-INF/**</exclude>
       </excludes>
     </fileSet>
-    <fileSet>
-      <directory>${project.build.directory}</directory>
-      <outputDirectory>jetty-base/lib/gae</outputDirectory>
```

Comment:
Do we need assembly.xml and the corresponding pluging execution at all anymore? It looks like everything is done by maven-resources-plugin and maven-dependency-plugin.